apisix/control/v1.lua

```diff
@@ -14,11 +14,33 @@
 -- See the License for the specific language governing permissions and
 -- limitations under the License.
 --
+local core = require("apisix.core")
+local plugin = require("apisix.plugin")
+
+
 local _M = {}
 
 
 function _M.schema()
-    return 200, {}
+    local schema = {
+        main = {
+            global_rule = core.schema.global_rule,
+            route = core.schema.route,
+            service = core.schema.service,
+            ssl = core.schema.ssl,
+            stream_route = core.schema.stream_route,
+            upstream = core.schema.upstream,
+        },
```

Comment:
how about could use `core.schema` as main ?

There will be redundant data, but no data will not be missed, and there is no need to modify the code here after adding a new schema.

Reply:
There is some methods in `core.schema` can't be json encoded, like `core.schema.check`.

Reply:
ok

Reply:
lack of consumer

Reply:
@nic-chen 
Solved.